### PR TITLE
hw-mgmt: scripts: fix removal of component in SMBIOS BOM string.

### PR DESCRIPTION
**Pull request by mshych**
Signed-off-by: Michael Shych <michaelsh@nvidia.com>

### CODE REVIEW

**Comment by mshych** [comment]
RERUN

File: usr/usr/bin/hw-management-devtree.sh [hunk 1]
```diff
@@ -305,66 +305,84 @@ devtr_check_board_components()
...
 		component_key=${comp:1:2}
-		# Don't process removed component
-		if [ "$component_key" == "0" ]; then
-			continue
-		fi
+
 		category=${category_arr[$category_key]}  # Optional, just for print.
 		case $category_key in
 			T)	# Thermal Sensors
+				# Don't process removed component
```
Comment (vadimp-nvidia) [inline usr/usr/bin/hw-management-devtree.sh:312]: Could you put '.' at the end of each comment in separate commit?

**Review by vadimp-nvidia — APPROVED** [review]

**Comment by sholeksandr** [comment]
RERUN

**Comment by mshych** [comment]
RERUN

**Comment by mshych** [comment]
RERUN

**Comment by mshych** [comment]
RERUN

**Comment by sholeksandr** [comment]
RERUN

**Comment by sholeksandr** [comment]
RERUN

**Pull request merged**